test/timestamp.py

```diff
@@ -126,4 +129,11 @@ def test_convert_timestamp(self):
         self.assertEqual(convert_timestamp("20171029010500Z"), (datet, date))
         self.assertEqual(convert_timestamp(date), (datet, date))
 
+    @mock.patch('vsc.utils.timestamp.read_timestamp')
+    def test_retrieve_timestamp(self, mock_read_timestamp):
```

Comment:
what happens when the file doesn't exist or contains gibberish?

Reply:
That's the responsibility of another function. Jens decided that if the file contains garbage, the `read_timestamp` returns `None` and is addressed in the conversion function, where it defaults to `today()`.

So, I think I'd better change that behaviour and convert the default if None is "read".

Reply:
i think we should fix that in an other PR, as it will affect other repos too

Reply:
can you add a test then that checks that if the file contains gibberish, the default timestamp is used instead?

Reply:
yes.